code/modules/cargo/packs.dm

```diff
@@ -543,6 +596,32 @@
 					/obj/item/ammo_box/magazine/wt550m9)
 	crate_name = "auto rifle ammo crate"
 
+/datum/supply_pack/security/armory/russian
+	name = "Russian Surplus Crate"
+	desc = "Hello Comrade, we have the most modern russian military equipment the black market can offer, for the right price of course. Sadly we couldnt remove the lock so it requires Armory access to open."
+	cost = 5000
+	contraband = TRUE
+	var/num_contained = 10
```

Comment:
new vars go @ bottom

Reply:
actually I don't see this being used to subtype (unlike `chef`), just hardcode the number in the fill proc?